### PR TITLE
Droth 679 admin class override in oth for admin class 1 road links

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala [hunk 1]
```diff
@@ -454,21 +454,29 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
     }
   }
 
+  def checkMMLId(vvhRoadLink: VVHRoadlink) : Option[Long] = {
+    vvhRoadLink.attributes.contains("MTKID") match {
+      case true => Some(vvhRoadLink.attributes("MTKID").asInstanceOf[BigInt].longValue())
+      case false => None
```
Comment (joanacoelho) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala]: There's a problem with that when the "MTKID" doesn't exist

**Comment by halied** [comment]
Ok

**Comment by marquesfm** [comment]
ok

**Comment by oskarmalmstrom** [comment]
ok

**Comment by marquesfm** [comment]
OK

**Pull request merged**